projects/sidekiqz2/system_bd.tcl

```diff
@@ -0,0 +1,263 @@
+ad_ip_parameter axi_ad9361_adc_dma CONFIG.AXI_SLICE_DEST 0
+ad_ip_parameter axi_ad9361_adc_dma CONFIG.DMA_2D_TRANSFER 0
+ad_ip_parameter axi_ad9361_adc_dma CONFIG.DMA_DATA_WIDTH_SRC 32
+
+ad_ip_instance util_fir_dec fir_decimator
+ad_ip_instance xlslice decim_slice
+
+# connections
+
+ad_connect  rx_clk_in axi_ad9361/rx_clk_in
+ad_connect  rx_frame_in axi_ad9361/rx_frame_in
+ad_connect  rx_data_in axi_ad9361/rx_data_in
+ad_connect  tx_clk_out axi_ad9361/tx_clk_out
+ad_connect  tx_frame_out axi_ad9361/tx_frame_out
+ad_connect  tx_data_out axi_ad9361/tx_data_out
+ad_connect  enable axi_ad9361/enable
+ad_connect  txnrx axi_ad9361/txnrx
+ad_connect  up_enable axi_ad9361/up_enable
+ad_connect  up_txnrx axi_ad9361/up_txnrx
+
+ad_connect  axi_ad9361/tdd_sync GND
+ad_connect  sys_200m_clk axi_ad9361/delay_clk
+ad_connect  axi_ad9361/l_clk axi_ad9361/clk
+
+ad_connect axi_ad9361/l_clk fir_decimator/aclk
+ad_connect axi_ad9361/adc_data_i0 fir_decimator/channel_0
+ad_connect axi_ad9361/adc_data_q0 fir_decimator/channel_1
+ad_connect axi_ad9361/adc_valid_i0 fir_decimator/s_axis_data_tvalid
+ad_connect axi_ad9361_adc_dma/fifo_wr_din fir_decimator/m_axis_data_tdata
+ad_connect axi_ad9361_adc_dma/fifo_wr_en fir_decimator/m_axis_data_tvalid
+ad_connect axi_ad9361/up_adc_gpio_out decim_slice/Din
+ad_connect fir_decimator/decimate decim_slice/Dout
+
+ad_connect axi_ad9361/l_clk fir_interpolator/aclk
+ad_connect axi_ad9361_dac_dma/fifo_rd_dout fir_interpolator/s_axis_data_tdata
+ad_connect axi_ad9361_dac_dma/fifo_rd_valid fir_interpolator/s_axis_data_tvalid
+ad_connect axi_ad9361/dac_valid_i0 fir_interpolator/dac_read
+ad_connect axi_ad9361_dac_dma/fifo_rd_en fir_interpolator/s_axis_data_tready
+ad_connect axi_ad9361/dac_data_i0 fir_interpolator/channel_0
+ad_connect axi_ad9361/dac_data_q0 fir_interpolator/channel_1
+ad_connect axi_ad9361/up_dac_gpio_out interp_slice/Din
+ad_connect fir_interpolator/interpolate interp_slice/Dout
+
+ad_connect  axi_ad9361/l_clk axi_ad9361_adc_dma/fifo_wr_clk
+ad_connect  axi_ad9361_adc_dma/fifo_wr_overflow axi_ad9361/adc_dovf
+ad_connect  axi_ad9361/l_clk axi_ad9361_dac_dma/fifo_rd_clk
+ad_connect  axi_ad9361_dac_dma/fifo_rd_underflow axi_ad9361/dac_dunf
+ad_connect  axi_ad9361/dac_data_i1 GND
+ad_connect  axi_ad9361/dac_data_q1 GND
+
+# interconnects
+
+ad_cpu_interconnect 0x79020000 axi_ad9361
+ad_cpu_interconnect 0x7C400000 axi_ad9361_adc_dma
+ad_cpu_interconnect 0x7C420000 axi_ad9361_dac_dma
+ad_mem_hp1_interconnect sys_cpu_clk sys_ps7/S_AXI_HP1
+ad_mem_hp1_interconnect sys_cpu_clk axi_ad9361_adc_dma/m_dest_axi
+ad_mem_hp2_interconnect sys_cpu_clk sys_ps7/S_AXI_HP2
+ad_mem_hp2_interconnect sys_cpu_clk axi_ad9361_dac_dma/m_src_axi
+ad_connect  sys_cpu_resetn axi_ad9361_adc_dma/m_dest_axi_aresetn
+ad_connect  sys_cpu_resetn axi_ad9361_dac_dma/m_src_axi_aresetn
+
+# interrupts
+
+ad_cpu_interrupt ps-13 mb-13 axi_ad9361_adc_dma/irq
+ad_cpu_interrupt ps-12 mb-12 axi_ad9361_dac_dma/irq
+
+
```

Comment:
How much of this file is directly copied from pluto? Does it makes sense to share a tcl between the two projects

Reply:
Mostly it's the same as pluto (only GPIO exported changes), but given that hardware is from different manufacturers I wouldn't share the TCL

Reply:
If then TCL is the same - why not share?

Reply:
Ok, will change design to source the pluto system_bd, and apply the modifications on top.